client/data/authorizations/actions.ts

```diff
@@ -110,6 +110,12 @@ export function* submitCaptureAuthorization(
 			'invalidateResolutionForStoreSelector',
 			'getAuthorizationsSummary'
 		);
+
+		yield dispatch(
```

Comment:
I was wondering if this invalidation would cause any side effects on the uncaptured table page. And vice-versa for the previous two invalidations. I didn't notice any side effects in my testing but want to double check with you.

Reply:
That does make sense, thanks for confirming :)